maestro/scripts/run_memverse_dot_com_tests.sh

```
@@ -0,0 +1,36 @@
+# Run the complete test flow
+echo -e "${YELLOW}Running create-login-delete flow test...${NC}"
+maestro test maestro/flows/memverse_dot_com/create_login_delete_flow.yaml --format=plain
+
+# Check the exit code
+if [ $? -eq 0 ]; then
+    echo -e "${GREEN}✓ All tests passed successfully!${NC}"
+    exit 0
+else
+    echo -e "${RED}✗ Tests failed. Please check the output above for details.${NC}"
+    exit 1
+fi</code_edit>
```

Shellcheck (0.10.0)

[error] Couldn't parse this redirection. Fix to allow more checks.

(SC1073)